watsononlinestore/watson_online_store.py

```diff
@@ -401,45 +499,46 @@ def format_discovery_response(response, data_source):
         if not ('results' in response and response['results']):
             return output
 
-        def get_product_name(entry, data_source):
+        AMAZON = "AMAZON"
+        IBM = "IBM_STORE"
+
+        def get_product_name(entry):
             """ Pull product name from entry data for nice user display.
 
-            :param str data_source: name of the discovery data source
             :returns: name of product
             :rtype: str
             """
             product_name = ""
 
-            if data_source == "amazon":
+            if data_source == AMAZON:
                 # For amazon data, Watson Discovery has pulled the
                 # product name from the html page and stored it as
                 # "title" in its enriched metadata that it generates.
                 if 'extracted_metadata' in entry:
                     metadata = entry['extracted_metadata']
                     if 'title' in metadata:
                         product_name = metadata['title']
-            elif data_source == "ibm_store":
+            elif data_source == IBM:
                 # For IBM store data, the product name was placed in
                 # text of the page, in the format:
                 # "Product: <product name> "Category".
                 if 'text' in entry:
-                    product_tag = "Product:"
-                    category_tag = "Category:"
+                    PRODUCT_TAG = "Product:"
+                    CATEGORY_TAG = "Category:"
```

Comment:
more constants to move to top. If you want them to stay local, then better to use lowercase. I see this is a grey area. Should look fine as constants an actually streamline this code a little.